.github/workflows/runtime-sync.yml

```diff
@@ -8,6 +8,11 @@ on:
     # Once per day at midnight PST (8 UTC)
     - cron: '0 8 * * *'
 
+permissions:
+  contents: write
+  issues: write
```

Comment:
Need `issues` permission to update #18943 when we're out-of-sync

Reply:
What does `contents` control? Branches?

Reply:
> What does `contents` control? Branches?

As far as we know, that's right. See discussion in dotnet/runtime#61626. I'm imitating that plus dotnet/runtime#61637 in this PR.